uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/directives/leave-request-popup.js

```diff
@@ -28,7 +29,8 @@ define([
                   return {
                     contactId: scope.contactId,
                     leaveRequest: scope.leaveRequest,
-                    sickness: scope.sickness
+                    sickness: scope.sickness,
+                    overtime: scope.overtime
```

Comment:
can we have a single variable here say `leaveType` which will take in a string for different types of leave like `sick`, `toil` ?